includes/blocks/class-llms-blocks-pricing-table-block.php

```diff
@@ -125,5 +140,3 @@ public function output( $attributes = array() ) {
 
 	}
 }
-
```

Comment:
we need to add all the public and protected methods inherited from the abstract class and deprecate them like we did with the syllabus.